src/tinypeel/Peeling/PeelingUpdates.py

```diff
@@ -111,6 +111,34 @@ def addIndividualToUpdate(d, p, LLp, LLpp):
     return LLp, LLpp
 
 
+def updateMafAfterPeeling(pedigree, peelingInfo):
```

Comment:
@RosCraddock @XingerTang the code loops over metafounders and for each metafounder loops over all pedigree members twice - once to calculate and then to assign anterior probabilities. I am thinking out loud if we can speed this up in any way if we flip iteration around - over individuals and then over metafounders, but we would be doing the same amount of work, so it seems we can not.

Reply:
Yes, I wondered the same when writing this!